content_type_test.go

```diff
@@ -536,8 +567,8 @@ func TestContentTypeFieldRef(t *testing.T) {
 	assertions := assert.New(t)
 
 	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		assertions.Equal(r.Method, "PUT")
-		assertions.Equal(r.RequestURI, "/spaces/"+spaceID+"/environments/"+environmentID+"/content_types/ct-name")
+		assertions.Equal(r.Method, "POST")
```

Comment:
If this is testing with the ID the request should be a PUT. The `Sys` object will need to be created here: https://github.com/regressivetech/contentful-go/blob/ca08c3e8fbd7a6066f6a4b01a8ea1b974fde2340/content_type_test.go#L617

Reply:
This is testing the type field ref validation. The content type with ID test is in line 503 and it is a PUT request there.